### PR TITLE
Update.readme

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

**Review by qrkourier — APPROVED** [review]
Happy to see the management API client hints too!

File: README.md [hunk 1]
```diff
@@ -1,21 +1,373 @@
...
+	
+	conn, _ := context.Dial(serviceName)
+	
+	if _, err := conn.Write([]byte("hello I am myTestClient")); err != nil {
+		panic(err)
+	}
+```
+
+### Load/Create A Configuration
+
```
Comment (plorenz) [inline README.md:100]: Stray ) after link

File: README.md [hunk 2]
```diff
@@ -1,21 +1,373 @@
...
+### Dial/Bind A Service
+
+The main activity performed with a [`Context`](ziti/contexts.go) is to dial or bind a service. In order for a dial or
+bind to be successful, the following must be true:
+
+1. The identity must have the proper dial or bind service policy to the service via [Service Policies](https://openziti.io/docs/learn/core-concepts/security/authorization/policies/overview#service-policies)
+2. The identity must have the proper dial or b ind services over at least one Edge Router via [Edge Router Policies](https://openziti.io/docs/learn/core-concepts/security/authorization/policies/overview#edge-router-policies)
+3. The service must be allowed to be dialed or bound on at least one Edge Router via [Service Edge Router Policies](https://openziti.io/docs/learn/core-concepts/security/authorization/policies/overview#service-policies))
+
+The easiest way to satisfy #2 and #3 are the make use of the `#all` role attributes when creating policies. Unless
```
Comment (plorenz) [inline README.md:164]: Second sentence is a fragment

File: README.md [hunk 3]
```diff
@@ -1,21 +1,373 @@
...
+    os.Exit(1)
+}
+```
+
+### Dial/Bind A Service
+
+The main activity performed with a [`Context`](ziti/contexts.go) is to dial or bind a service. In order for a dial or
+bind to be successful, the following must be true:
+
+1. The identity must have the proper dial or bind service policy to the service via [Service Policies](https://openziti.io/docs/learn/core-concepts/security/authorization/policies/overview#service-policies)
```
Comment (plorenz) [inline README.md:160]: b ind

**Review by plorenz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (andrewpmartinez) [reply README.md]: Fleshed it out.

**Pull request merged**